src/main/flight/servos.c

```diff
@@ -480,7 +481,12 @@ void servoMixer(void)
     }
 
     for (int i = 0; i < MAX_SUPPORTED_SERVOS; i++) {
-        servo[i] = ((int32_t)servoParams(i)->rate * servo[i]) / 100L;
+        int32_t rate = (int32_t)servoParams(i)->rate;
+        if (IS_RC_MODE_ACTIVE(BOXSERVOCUSTOMRATE)) {
```

Comment:
IMO much better  location is Rate profile (and applying only to flight-related servos).

This can be (partially) handled by allowing current rate profile as mixer condition above.

Reply:
but that would mean that servo code will ahve to access the rates code? is that ok?

And yeah i can probably make it apply only to flight-related servos.

Reply:
> but that would mean that servo code will have to access the rates code? is that ok?

There should be some way to adjust wing flight behavior in rate profiles. But simply stuffing another parameter into rate profile is quite ugly.